src/client/components/Nav/index.js

```diff
@@ -20,7 +20,9 @@ const Nav = ({
     });
   };
 
-  // useEffect(addAnalytics('research'));
+  window.onload = function() {

```

Comment:
@finnzink You should add a comment on why this is needed and why useEffect didn't work.

Reply:
@rlorenzo The reason useEffect doesn't work is too long for an in-code comment, but basically there are two issues. First, it runs three times (I assume for each layer of the component tree) which registers three clicks. There is an additional error: state update on unmounted component. I think this is because the async calls in the api call take longer than the useEffect function and it can't clean up. Not exactly sure.

Reply:
@maxpluez @kilometerskang @chern Any ideas on better workarounds or is it common to use window.onload in a React app?

@finnzink The comment, even if long, is needed to explain to future developers why we are breaking out of the React framework to do this.

Reply:
If im not mistaken, the 2nd parameter in useEffect can affect how many times the function gets called, and I think if you set it to an empty array [], it should be called once.

Reply:
@rlorenzo here's a workaround which works: 
```  
const addAnalytics = (type = 'research') => {
    ltikPromise.then(ltik => {
      axios.get(`/api/addanalytics/${type}?ltik=${ltik}`).then(res => {});
    });
  };

useEffect(addAnalytics, []);
```
Basically adding a default param in addAnalytics. No idea why passing a param breaks everything. Is this ok?

Reply:
@kilometerskang You are right. A quick search came up with this: https://stackoverflow.com/questions/57597253/useeffect-hook-executes-multiple-times-on-updating-state